openwisp_controller/connection/tests/test_api.py

```diff
@@ -410,6 +417,18 @@ def test_filter_credentials_list(self):
         self.assertContains(response, cred_2.id)
         self.assertNotContains(response, cred_1.id)
 
+    @patch.object(ListViewPagination, "page_size", 3)
+    def test_credential_list_ordering(self):
```

Comment:
This could be a subtest of `test_get_credentials_list`, can you please merge this with that test and simplify the logic a bit to avoid bloat? Just create 3 credentials, check if you can set the creation date of one of them manually to an older date and verify that it's shown as last.